pkg/shardddl/optimism/lock.go

```diff
@@ -28,6 +28,15 @@ import (
 	"github.com/pingcap/dm/pkg/terror"
 )
 
+const (
```

Comment:
maybe define a type for it, because when I see the value of map is `int` I assume that it means the count of not dropped columns

OK to not change if need change too many place